gensim/models/tfidfmodel.py

```diff
@@ -65,23 +117,69 @@ def __init__(self, corpus=None, id2word=None, dictionary=None,
 
         so you can plug in your own custom `wlocal` and `wglobal` functions.
 
-        Default for `wlocal` is identity (other options: math.sqrt, math.log1p, ...)
-        and default for `wglobal` is `log_2(total_docs / doc_freq)`, giving the
-        formula above.
 
-        `normalize` dictates how the final transformed vectors will be normalized.
-        `normalize=True` means set to unit length (default); `False` means don't
-        normalize. You can also set `normalize` to your own function that accepts
-        and returns a sparse vector.
+        Parameters
+        ----------
+        corpus :    dictionary.doc2bow
+                    Corpus is a list of sets where each set has two elements. First being the termid and
+                    second being the term frequency of each term in the document.
+        id2word :   dict
+                    id2word is an optional dictionary that maps the word_id to a token.
+                    In case id2word isn’t specified the mapping id2word[word_id] = str(word_id) will be used.
+        dictionary :corpora.Dictionary
+                    If `dictionary` is specified, it must be a `corpora.Dictionary` object
+                    and it will be used to directly construct the inverse document frequency
+                    mapping (then `corpus`, if specified, is ignored).
+        wlocals :   user specified function
```

Comment:
Instead of
```
        wlocals :   user specified function
                         Default for `wlocal` is identity (other options: math.sqrt, math.log1p, ...)
```

should be
```
        wlocals : function, optional
            description of parameter, with links to different implementations (if function defined in gensim, link should be like :func:`~gensim.some.func`)
```

everywhere